addons/binding/org.openhab.binding.sleepiq/src/main/java/org/openhab/binding/sleepiq/internal/SleepIQConfigStatusMessage.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.sleepiq.internal;
+
+/**
+ * The {@link SleepIQConfigStatusMessage} defines the keys to be used for configuration status messages.
+ *
+ * @author Gregory Moyer - Initial contribution
+ *
+ */
+public enum SleepIQConfigStatusMessage {
+
+    USERNAME_MISSING("missing-username-configuration"),
```

Comment:
I followed this pattern from the Hue binding in ESH (see HueConfigStatusMessage). Please advise if this pattern should not be followed.